manifests/vanilla/deploy-csi-snapshot-components.sh

```diff
@@ -0,0 +1,95 @@
+#!/bin/bash
+# Copyright 2021 The Kubernetes Authors.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#    http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -e
+
+if [ "$1" = "-h" ] || [ "$1" = "--help" ]; then
+    cat <<EOF
+Usage: Deploys the Snapshot Controller which is necessary for Snapshot feature
+
+1. Deploys the CSI Snapshot CRDs
+2. Creates RBAC rules to support CSI Snapshot
+3. Deploys snapshot-controller in kube-system namespace
+
+Refer https://kubernetes-csi.github.io/docs/snapshot-controller.html for further information.
+
+Example command:
+
+./deploy-snapshot-controller.sh --release v4.1.1
```

Comment:
Do we want to call out the highest snapshot-controller/external-snapshotter version that is compatible with vSphere CSI snapshot alpha release, like, v4.1.1, here? cc: @xing-yang